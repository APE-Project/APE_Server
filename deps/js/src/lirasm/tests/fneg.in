p = immd 123.456
n = negd p
retd n
