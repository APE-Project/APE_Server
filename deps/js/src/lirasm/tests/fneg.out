Output is: -123.456
